Dockerfile

```diff
@@ -36,6 +36,8 @@ RUN \
   apt-get install --no-install-recommends -y libxtst6 xvfb xauth x11-apps imagemagick zip; \
   `# libraries needed by JCEF` \
   apt-get install --no-install-recommends -y libatk1.0-0 libatk-bridge2.0-0 libgbm1 libglib2.0 libxcomposite1 libxkbcommon0 libxrandr2; \
+  useradd -m user; \
```

Comment:
Hmmm but note that without `USER` instruction in Dockerfile, the code in the container will still run as root user 🤔

And hence the files created within the container will be owned by root on the host... can you actually observe this behavior on Mac OS? maybe the mechanism is different there

Reply:
> is owned by this user

Okay... but I suppose the UID/GID of the user in Docker container and of the one in your machine are different nevertheless? Pls check with `id -a` in both places

Reply:
in what two places ? I do not have this `user` on my machine (I probably missed something). I can only check UID/GID in container or directly while creating the image

Reply:
Just run:

```
$ id -a
$ docker run <your-image> id -a
```

Reply:
The user **names** don't matter, as long as their numeric **ids** are the same!

Reply:
I suppose your host 503:20 (uid:gid) user can**not** write files created by container 1000:1000 user, right?

Reply:
you mean exec into created container as my host 503:20 ? if so then I am not able to do it

Reply:
I mean sth like that:
```
docker run -v <some-local-dir>:/foo <your-image> touch /foo/bar.txt
```

Now `<some-local-dir>/bar.txt` exists on the host, but is owned by 1000:1000... and will have `rw-r--r--` permissions, hence your local user (503:20) won't be able to write to this file.

Pls check if this holds true on Mac!

Reply:
if I run container in such a way: 
` docker run -v "PWD":/foo gitmachete/intellij-plugin-ci:6.2.1 /bin/bash -c "touch /foo/bar.txt; while true; do sleep 1; done"`
foo.txt is owned by root (because docker run execute as root) and when I e.g. execute `docker run -u user` and run uiTest, artifacts are stored inside `/home/user/artifacts/` and they are owned by `user`

Reply:
and as local user (503:20) I cannot event exec into container (because such user not exist in image)

Reply:
I will try to rework as in #964

Reply:
Discussed on Slack... probably Docker for Mac has some mapping of UID/GID such that every file created from container in the shared volume, on host belongs just to the host user (and not to whatever UID/GID were used in the container)